ledger.go

```diff
@@ -154,6 +163,29 @@ func EvmosLedgerDerivation(config params.EncodingConfig) LedgerDerivation {
 	}
 }
 
+// Helper function to display the EIP-712 hashes; this allows users to verify the hashed message
+// they are signing via Ledger.
+func (e EvmosSECP256K1) displayEIP712Hash(typedData apitypes.TypedData) error {
+	domainSeparator, err := typedData.HashStruct("EIP712Domain", typedData.Domain.Map())
+	if err != nil {
+		return err
+	}
+	typedDataHash, err := typedData.HashStruct(typedData.PrimaryType, typedData.Message)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Signing the following payload with EIP-712:\n")
+	fmt.Printf("- Domain: %v\n", bytesToHexString(domainSeparator))
+	fmt.Printf("- Message: %v\n", bytesToHexString(typedDataHash))
+
+	return nil
+}
+
+func bytesToHexString(bytes []byte) string {
+	return "0x" + strings.ToUpper(hex.EncodeToString(bytes))
+}
```

Comment:
You can use the `common.Bytes2Hex` function instead

Reply:
Same implementation:

```
// Bytes2Hex returns the hexadecimal encoding of d.
func Bytes2Hex(d []byte) string {
	return hex.EncodeToString(d)
}
```